Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1464,12 +1464,6 @@ def generate_message_names_header(receivers):
         result.append('    return Detail::messageDescriptions[static_cast<size_t>(messageName)].%s;\n' % fname)
         result.append('}\n')
         result.append('\n')
-    result.append('inline ASCIILiteral descriptionLiteral(MessageName messageName)\n')
-    result.append('{\n')
-    result.append('    messageName = std::min(messageName, MessageName::Last);\n')
-    result.append('    return ASCIILiteral::fromLiteralUnsafe(Detail::messageDescriptions[static_cast<size_t>(messageName)].description);\n')
-    result.append('}\n')
-    result.append('\n')
     result.append('constexpr bool messageIsSync(MessageName name)\n')
```

Comment:
It seems like the `webkitpy` tests might be angry after this change. There might be an expectation file that needs to be updated?